### PR TITLE
Minesweeper for iOS and Web

### PR DESCRIPTION
**Pull request by serbelga**
## Run on iOS simulator
```
./gradlew iosDeployIPhone13ProMaxDebug
./gradlew iosDeployIPadDebug
```

## Run on Web
```
./gradlew :web:jsBrowserRun
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # :bomb: Minesweeper :triangular_flag_on_post:
 The classic Minesweeper game for Android with the ol' good Windows 3.1 appearance, entirely written with :sparkles:Jetpack Compose:sparkles:.
```
Comment (Xacalet) [inline README.md]: I guess it's not only for Android anymore :)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
 
+## Run on iOS simulator
+`./gradlew iosDeployIPhone13ProMaxDebug`
+`./gradlew iosDeployIPadDebug`
+
+## Run on Web
+`./gradlew jsBrowserRun`
+
 ## 🚧 TODO 🚧
```
Comment (Xacalet) [inline README.md:11]: Maybe you could add here what's missing in the iOS version (icons, typography, etc ...)

File: common/build.gradle.kts [hunk 3]
```diff
@@ -8,20 +6,26 @@ plugins {
 }
 
 group = "com.xacalet.minesweeper.common"
+version = "1.0"
 
 kotlin {
     android()
     jvm("desktop")
+    iosX64()
+    iosArm64()
```
Comment (Xacalet) [inline common/build.gradle.kts]: Why not using `ios()`. Won't it include both `iosX64()` and `iosArm65()`?

File: common/build.gradle.kts [hunk 4]
```diff
@@ -40,9 +46,31 @@ kotlin {
...
+        val iosArm64Test by getting
+        val iosSimulatorArm64Test by getting
+        val iosTest by creating {
+            dependsOn(commonTest)
+            iosX64Test.dependsOn(this)
+            iosArm64Test.dependsOn(this)
+            iosSimulatorArm64Test.dependsOn(this)
+        }
+        val jsMain by getting
+        val jsTest by getting
```
Comment (Xacalet) [inline common/build.gradle.kts]: By adding `ios()` as target, I guess you can get rid of the specific-architecture sourcesets, and generate the one related to iOS by doing `val iosMain by getting`

File: README.md (hunk 1 above)
Comment (serbelga) [reply README.md]: Updated 😄

File: common/build.gradle.kts (hunk 3 above)
Comment (serbelga) [reply common/build.gradle.kts]: Resolved

File: common/build.gradle.kts (hunk 4 above)
Comment (serbelga) [reply common/build.gradle.kts]: Done

**Review by Xacalet — APPROVED** [review]

**Pull request merged**